src/Features/CSharp/Portable/Completion/KeywordRecommenders/LoadKeywordRecommender.cs

```diff
@@ -3,6 +3,7 @@
 using System.Threading;
 using Microsoft.CodeAnalysis.CSharp.Extensions;
 using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
+using Microsoft.CodeAnalysis.Shared.Extensions;
```

Comment:
Yes. The `IsInteractiveOrScript` was already part of CSharp specific extensions. After moving the method to shared extensions the new using was necessary.